PlayCover/AppInstaller/Installer.swift

```diff
@@ -8,8 +8,35 @@
 import Foundation
 
 class Installer {
+
+    static func installPlayToolsPopup() -> Bool {
+        let alert = NSAlert()
+        alert.messageText = NSLocalizedString("alert.install.injectPlayTools", comment: "")
+        alert.informativeText = NSLocalizedString("alert.install.injectPlayToolsInApp", comment: "")
+
+        alert.showsSuppressionButton = true
+        alert.suppressionButton?.toolTip = NSLocalizedString("alert.supression", comment: "String")
+
+        let yes = alert.addButton(withTitle: NSLocalizedString("button.Yes", comment: ""))
+        alert.addButton(withTitle: NSLocalizedString("button.No", comment: ""))
+
+        yes.hasDestructiveAction = true
+
+        let response = alert.runModal()
+
+        if alert.suppressionButton?.state == .on {
+            InstallSettings.shared.showInstallPlayToolsPopup = false
+            InstallSettings.shared.alwaysInstallPlayTools = response == .alertFirstButtonReturn
```

Comment:
This option feels largely redundant. If you always install PlayTools, why ask the user whether or not to install it? It feels like the show PlayTools popup shouldn't appear by default, and shouldn't be available if this option is set to true.

Reply:
The `alwaysInstallPlayTools` is only used when the user suppresses PlayTools install popup. When it is suppressed, it installs PlayTools based on the option they chose when they suppressed the popup. When the popup is used, it installs PlayTools based on what the user choses.

Reply:
Maybe the `alwaysInstallPlaytools` option should be made disabled when `showInstallPlaytoolsPopup` is enabled to make its functionality clearer.